LiskHQ/lisk-sdk
# Fix generator enable/disable commands

### What was the problem?

This PR resolves #7783 

### How was it solved?

- Revert generator enable/disable command to have `--use-status-value`
  - Move height/max-height-generated/max-height-prevoted as flag
- Revert temporally lisk-db version to the previous version because it leaves locks value https://github.com/LiskHQ/lisk-db/issues/78
- Fix to pass `keypairs` from `generator` to `endpoint`
- Remove keypair when setting new key

### How was it tested?

1. clean ` rm -rf ~/.lisk/pos-mainchain`
2. encrypt keys `./bin/run keys:encrypt -f ./config/default/dev-validators.json`
3. import new keys `./bin/run keys:import -f config/default/dev-validators.json`
4. enable block generation `./bin/run generator:enable lsk2xxvfxaqpm42wr9reokucegh3quypqg9w9aqfo --height=0 --max-height-prevoted=0 --max-height-generated=0`
5. check block is generated (wait for 1 round)
6. check status `./bin/run generator:status --pretty`
